test/test_JSON/test_json_wrong_device.json

```diff
@@ -0,0 +1 @@
uvaTlTQSUZXTJgnisQ+FmJnFA\nrD390NweDls=\n-----END CERTIFICATE-----\n","certificate":"-----BEGIN CERTIFICATE-----\nMIIC+TCCAeGgAwIBAgIJDm6sugeAT2REMA0GCSqGSIb3DQEBCwUAMBYxFDASBgNV\nBAMMC3dhcHBzdG8uY29tMB4XDTIwMDEyMDA5MTgyNloXDTIzMDExOTA5MTgyNlow\nLzEtMCsGA1UEAwwkYjAzZjI0NmQtNjNlZi00NDZkLWJlNTgtZWYxZDFlODNiMzM4\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4SEBBdKcs1l4iTrIIxF2\nQ6tRUcpVR+19CVyFRhXOf4svSHhd20a8DB3A1u/oEjaS4jtcWwYDRdTSY5GVK0AY\n/tOkiPIbuWhOO37uF+FvOxjLxPWI8hOX4AJgvgi33i7FhxpBzcWe/L1scq0VcBro\nPgvoB5GOYhqDbk4QIfFAzw3/mT/p2hlLWRN38YH0vuws9aKH/qk69/paRNZYlwuS\nCOESs/viU9GWGfuBRM66ckmVSXIbzUL5eUUAn1hREg276Ikzc/8wirqwnLQdmv36\n0frdKHS0jn2RrSqE8BEouOHvVdUyfc8SdxxX1a5m6vwqSeZELc+zU+EySZPMEeyJ\newIDAQABozEwLzAOBgNVHQ8BAf8EBAMCB4AwHQYDVR0OBBYEFBFr2bBOTQzuW4Ky\nw8haRcNf5itnMA0GCSqGSIb3DQEBCwUAA4IBAQAlw81eN3xuMuA6eOeg1W7t8shs\n6/kh/uLh5uKnWtVntPUt36l24DN0l0t0mAMILnwunW0gdSfROs+SecE/6XOLKoAq\ni5GTBqult6uaEpjbgpcxeH7XBe0Z8JlMUQ3dCWsOPqlHuBuBjf8SGWRTcjNFy4Np\niDlkc4IKe/fL2Lv6cnguIouc9gG8tFB5FgcokSoQWDLtqd9Lrq8l1jq+8Q9iy7wL\nFksY3xd/UKQ3ERNnj9zRAv60X8a+usm0UpVlGQRWBm49xt2v688ATbbfUxfMOsAp\nAjfhBPoQhlc7WK6hp++n1WHGXym0/nf3A+vBFoRpJYc2rsXDFsxDOSrAF7bk\n-----END CERTIFICATE-----\n","private_key":"-----BEGIN RSA PRIVATE KEY-----\nMIIEowIBAAKCAQEA4SEBBdKcs1l4iTrIIxF2Q6tRUcpVR+19CVyFRhXOf4svSHhd\n20a8DB3A1u/oEjaS4jtcWwYDRdTSY5GVK0AY/tOkiPIbuWhOO37uF+FvOxjLxPWI\n8hOX4AJgvgi33i7FhxpBzcWe/L1scq0VcBroPgvoB5GOYhqDbk4QIfFAzw3/mT/p\n2hlLWRN38YH0vuws9aKH/qk69/paRNZYlwuSCOESs/viU9GWGfuBRM66ckmVSXIb\nzUL5eUUAn1hREg276Ikzc/8wirqwnLQdmv360frdKHS0jn2RrSqE8BEouOHvVdUy\nfc8SdxxX1a5m6vwqSeZELc+zU+EySZPMEeyJewIDAQABAoIBAERlqZDiTvdyXFuc\n/0cgPAUNHJSz2zsSHSXl9Ru47lbzq3biEunW0LUBqR/kHWbDK9cN395LslzZ7t2P\nUnmkbLOnC1sL635mATa0NpQMyqVywB3j1yps53yIXGn/7hTTsvF7+bgRrV7uEe+5\nREUd96c6KNF72H1XvsyZbO9TA7Sj4xTOgHtm4vvAwVZr1+d0lu3Ojuh4KVamW6wm\nvkkZV386OTupTwBkNB6c+EMLdVln7v/CjKgHvogDPSkd9OvNK+RHlgZSUYP236kQ\nZJShlwhiD6VYBmdAqbfjBXDs2wVo3zV5OEh17iYVQ8dodATYZMWKXGrS+UI8cLvO\ncbJLVFECgYEA+8QeYRY/bc82ar02LA9nlgJSxUCjOuvqimvdoA1N9F4Z5sEF8V4J\nQ9bJOVRDjP4cVFkUt1eBYU/7dfwIlhyRyJEbDyrO+6tH43nMZiF6fMVaGNWZ5hJF\nXPhqdDmEDJswS06WjWToGQH6NMHoXVQtA6SL03LQWXVv4dlsfZsI/PMCgYEA5Oo1\nlnjZpFDRwwQQ+IlMyM486mtWediPRjEQWMeQHpDzkxdTqW0ouyQh2eG1xLh/El/A\nru8zSJQH4/CHxkArqUPrcbngnA+rvOYDP6HbfdF4grVEn+z5+jLbbx/SgDtRfI1d\nm7Pn9OSgNCsu/45/hK82iyZPxZjJPr5PbWpDQ1kCgYAIQW0Rl+4qhqHtLBkrprGp\n3JekOjNgCj5dnZ3d0rHcsn088NVmDcRWAiQFCjtQ00BQj787J9ZpP3sqakJJ0dQB\n9Kgjfi7dV8d6sRK/wpdGoh4LNG3genblzouHFh+7XhdeUMxlBIsLzuHKexVIgisb\ntHiVTW2BkXVCvaNuFvhbewKBgQDcyXDKIuF39GN8vq1HqdWfUBu81a5gcxktUCE+\nH6zHbnMfib4tiZVhemYT6fkRNgmJxUTo9Nj3xZ8eoF20PyxdY57w36DdpNe1p/u2\nIfy7i4+6Me9weHZadTbkJUM4Xv4nLYn9pTWyI9jIuGXqIpyncv8ekuMZ6FUh6Rhe\npcEecQKBgEfvrIXhrT5JIIIXpYY42Xqqwt2m7ecZjhOGjz+g99ISgrRfwD636q6h\nkpNZaI3XqCi6cn67uXDz5ZNghZC3NIagKrtVZkxm84rEF/LJOHmqR9ndEdvCQdQR\nhUDmrfuIP51rd1nOz4I2S/HBVKWdGgloyZ9HEa8KJz/iy2qx/1xk\n-----END RSA PRIVATE KEY-----\n","meta":{"id":"6b1b974a-7c47-4d04-9e49-9baaea4bacb4","type":"creator","version":"2.0"}}
```

Comment:
You do not need a valid certificate in the json file. Just write some dummy data.